src/routes/changelog/(entries)/2025-03-21.markdoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: changelog
+title: Backup performance improved by over 700%
+date: 2025-03-21
+---
+
+Appwrite Cloud backups are now significantly faster, reducing wait times and improving reliability.
+
+## What's improved
+
+- **Backup creation** is **7x faster** (from **241.3s** → **29.1s**)
+- **Backup archiving** is nearly **3x faster** (from **5.8s** → **1.5s**)
+- **Overall backup process** is **7x faster** (from **247.1s** → **30.6s**)
```

Comment:
Let's remove the seconds values, they're specific to the test dataset

Reply:
Done!